client/src/containers/MathBoxScene/components/MathBoxScene.js

```diff
@@ -27,6 +33,23 @@ type Props = {
   renderErrors: ErrorState,
   setError: SetError
 }
+
+export const mathboxElement = document.getElementById("mathbox");
```

Comment:
okay, the next two `const`s are the meat. Do we want these declared somewhere else?